dist/stylesheet.css

```diff
@@ -14,18 +30,128 @@
   cursor: pointer;
 }
 
-.t_search-box-overlay {
-  background-color:white;
+.t_search-box-container {
+  background-color:#fff;
   z-index: 9;
   cursor: initial;
 }
 
+.t_search-form {
+  position: relative;
+  line-height: 1.5rem;
+}
+
+.t_search-field {
+  position: relative;
+  max-width: 88rem;
+  padding: .5rem 0;
+  margin: 0 1rem;
+}
+
+.t_search-field::before {
+  position: absolute;
+  top: 50%;
+  left: 1rem;
+  width: 1.5rem;
+  height: 1.5rem;
+  pointer-events: none;
+  content: "";
+  background: url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' height='24' width='24'%3E%3Cpath d='M21 19.6l-5.4-5.4C16.5 13 17 11.6 17 10c0-3.9-3.1-7-7-7s-7 3.1-7 7 3.1 7 7 7c1.6 0 3-.5 4.2-1.4l5.4 5.4 1.4-1.4zM10 15c-2.8 0-5-2.2-5-5s2.2-5 5-5 5 2.2 5 5-2.2 5-5 5z'/%3E%3C/svg%3E");
+  transform: translateY(-50%);
+}
+
+.t_input-bar {
+  display: block;
+  width: 100%;
+  padding: .75rem 7.25rem .75rem 3.25rem;
+  margin: 0 auto;
+  font-size: 16px;
```

Comment:
"rem" references ONLY the font size of the root <html> element, not the current element. "em" references font of the parent element. Assuming your elements are just naturally inheriting the root 16px font size, you shouldn't need to specify it everywhere.